compiler/compiler.go

```diff
@@ -3219,8 +3221,8 @@ func (c *Compiler) ExternalInt64AsPtr() error {
 			entryBlock := llvm.AddBasicBlock(externalFn, "entry")
 			c.builder.SetInsertPointAtEnd(entryBlock)
 			var callParams []llvm.Value
-			if fnType.ReturnType() == int64Type {
-				return errors.New("todo: i64 return value in exported function")
+			if fnType.ReturnType() == int64Type && !c.Config.WasmI64Enable {
+				return errors.New("i64 return value in exported functions disallowed by default, use --wasmi64enable for non-JS wasm targets")
```

Comment:
This change is not necessary. With the `-wasmi64enable` flag, this function won't be called anyway.
The same goes for `WasmI64Enable` in `compiler.Config`.